### PR TITLE
Create new study

### PR DESCRIPTION
**Pull request by ChamseddineBhd**

### CODE REVIEW

File: src/components/directory-tree-view.js [hunk 1]
```diff
@@ -1,31 +1,100 @@
 /**
- * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (AbdelHedhili) [inline src/components/directory-tree-view.js]: why go back to 2020 ?

File: src/components/directory-tree-view.js [hunk 2]
```diff
@@ -1,31 +1,100 @@
...
+    );
 
     const selectedDirectory = useSelector((state) => state.selectedDirectory);
 
+    const dispatch = useDispatch();
+    const intl = useIntl();
+
+    const handleOpenMenu = (event) => {
+        setAnchorEl(event.currentTarget);
+        event.stopPropagation();
```
Comment (AbdelHedhili) [inline src/components/directory-tree-view.js:68]: why is this needed ?

File: src/translations/en.json [hunk 3]
```diff
@@ -10,5 +10,23 @@
...
+    "caseName": "case name",
+    "uploadCase": "Upload case",
+    "uploadErrorMsg": "Please upload the case file",
+
+    "uploadMessage": " No file selected",
+    "cancel": "CANCEL",
+    "create": "CREATE",
+    "newStudy": "Add new study",
+    "creationInProgress": "creation in progress...",
+    "studyCreatingError": "An error occurred while creating the study : {studyName}",
```
Comment (AbdelHedhili) [inline src/translations/en.json]: "An error occurred while creating the study: {studyName}" (no space before ":")

**Review by AbdelHedhili — APPROVED** [review]

File: src/components/directory-tree-view.js (hunk 2 above)
Comment (ChamseddineBhd) [reply src/components/directory-tree-view.js:68]: to avoid event propagation the higher order elements
